Traktor Kontrol S2MK3: Add documentation for PR#11702 (#568)
mixxxdj/manual: source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

| buttons            |                                                                                                                             |                                                     |
+--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
| Hotcue mode        | If hotcue is set, seeks the player to hotcue position. Otherwise set hotcue at current position                             | Clear the hotcue                                    |
+--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
| Samples mode       | If track is loaded into corresponding slot, go to CUE point and play                                                        | If track is playing, CUE default behaviour.         |

> |                    |                                                                                                                             | Otherwise eject track                               |

Reviewer: Does pressing the button load the selected track?
Just wondering, because Shift+press can eject, and if that can be done why not also load.

> | Samples mode       | If track is loaded into corresponding slot, go to CUE point and play                                                        | If track is playing, CUE default behaviour.         |
> |                    |                                                                                                                             | Otherwise eject track                               |

Reviewer: Yes, press without Shift does load the selected track (if none is loaded).
Please add that info here:
```suggestion
| Samples mode       | Load selected track into corresponding slot                                                                                 | If track is playing, CUE default behaviour.         |
|                    | If track is loaded into corresponding slot, go to CUE point and play                                                        | Otherwise eject track                               |
```

Reviewer: @olivier-mauras A note: we're now in the final release phase now and would love to get this merged soonish.
If you don't respond or simply don't find time to adopt this change (squash and force-push) in the next let's say two days I'll take the liberty to push this change myself.

Author: @ronso0 I'm quite overloaded currently and won't have time to take care of this in the coming weeks, so please go ahead.

Reviewer: Okay, thanks for your quick response!

+------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
| (25) LOW knob    | Low frequency filter                                                                                                        |                                                     |
+------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+

> | 26. Effect knob  | Quick effect knob for the corresponding deck                                                                                |                                                     |

Reviewer: Let's stick the current numbering format (here and below)
```suggestion
| (26) Effect knob  | Quick effect knob for the corresponding deck                                                                                |                                                     |
```

Reviewer: Oh, the current version is already a mix of 12. and (13).
Do you mind unifying that? I'd prefer 12., or just a bold number **12**, maybe we don't need a . or () then.

Reviewer: *ping*

Author: No I don't mind unifying, which one then? :)

Reviewer: Either the numbers are in a separate column (too much hazzle if yo ask me), or 12. or **12**. I'd try both (`make html`) then choose what you prefer : )

Author: **12**. has my preference so let me fix that ;)

| (29) Headphone   | Adjusts the headphone output gain                                                                                           |                                                     |
| gain knob        |                                                                                                                             |                                                     |
+------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+

> | (30) Effect      | | Enable effect from the quick effect presets list on both decks. 8 first effects from the list can be selected.            | Enable effect on selected deck                      |
> | buttons          | | Press the button once to get the first effect, press twice for the second effect.                                         |                                                     |

Reviewer: Let's use the correct terms
```suggestion
| (30) Effect      | | Load preset from the Quick Effect presets list on both decks. 8 first presets from the list can be selected.            | Load preset on selected deck                      |
| buttons          | | Press the button once to get the first preset, press twice for the second preset.                                         |                                                     |
```

| (29) Headphone   | Adjusts the headphone output gain                                                                                           |                                                     |
| gain knob        |                                                                                                                             |                                                     |
+------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+

> | 30. Effect       | | Enable effect from the quick effect presets list on both decks. 8 first effects from the list can be selected.            | Enable effect on selected deck                      |

Reviewer: What's the timeout to make the "second" press a first press?

Reviewer: I get what you want to illustrate with that table, but it's not clear if I can press 1, then 2 to get effect 6?

What do you think about this
```
1st press (2nd press)

1 (5)  2 (6)
3 (7)  4 (8)
```

Reviewer: Oh, got it:
pressing the button  when the preset is already selected selects index+4

I'd say the description can be improved.

Author: > I get what you want to illustrate with that table, but it's not clear if I can press 1, then 2 to get effect 6?

No you can't, if you want effect 6 you need to press twice on button 2

> I'd say the description can be improved.

Yeah maybe the keyword should be "press twice to get second effect"
